.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+# See the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "gomod"
+    directory: "/tools"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "github-actions"
```

Comment:
nitpick: can we put this one on top or bottom, but not between go and rust? That way, we have infra / code part if that makes sense?

Reply:
Even threw some comments to make it even better